website/project/views/node.py

```diff
@@ -247,6 +247,14 @@ def node_forks(auth, node, **kwargs):
 @must_be_contributor
 def node_setting(auth, node, **kwargs):
 
+    #check institutions:
+    try:
+        inst = Institution.find_one(Q('email_domain', 'eq', auth.user.username.split('@')[1]))
+        auth.user.affiliated_institutions.append(inst)
```

Comment:
@hmoco Is it possible for the same institution to be added more than once here?

Also, is this automatic affiliation the desired behavior? @icereval